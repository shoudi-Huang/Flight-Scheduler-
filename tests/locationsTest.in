locations
location add NewDelhi 28.527 77.0688988 -0.123
location add Mumbai 19.08 72.741 0.4
LOCAtionS
location add Berlin 52.5 13.15 0.22222
location add NewYork 40.6971494 -74.2598661 -0.09853
locations a 10
exit
