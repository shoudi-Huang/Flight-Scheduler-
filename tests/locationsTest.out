User: Locations (0):
(None)

User: Successfully added location NewDelhi.

User: Successfully added location Mumbai.

User: Locations (2):
Mumbai, NewDelhi

User: Successfully added location Berlin.

User: Successfully added location NewYork.

User: Locations (4):
Berlin, Mumbai, NewDelhi, NewYork

User: Application closed.
